faq.rst

```diff
@@ -0,0 +1,46 @@
+Frequently Asked Questions
+==========================
+
+The following FAQs are organized into
+
+- a "Components" section that includes each of the seven PURL components
+  (``scheme``, ``type``, ``namespace``, ``name``, ``version``, ``qualifiers``
+  and ``subpath``), and
+
+- a "General" section containing a mix of questions and answers that don't fit
+  neatly into a component-focused category.
+
+If you have a question about the PURL specification and don't find an answer
+below, you can open an issue `here <https://github.com/package-url/purl-spec/issues/new?template=Blank+issue>`_.
+
+Components
+~~~~~~~~~~
+
+Scheme
+------
+
+**QUESTION**: Can the ``scheme`` component be followed by a colon and two slashes, like a URI?
+
+No.  Since a Package-URL, or PURL, never contains a URL Authority, its ``scheme`` should not be suffixed with double slash as in 'pkg://' and should use 'pkg:' instead. Otherwise this would be an invalid URI per RFC 3986 at https://tools.ietf.org/html/rfc3986#section-3.3::
```

Comment:
```suggestion
No. Since a ``purl`` never contains a URL Authority, its ``scheme`` should not be suffixed with double slash as in 'pkg://' and should use 'pkg:' instead. Otherwise this would be an invalid URI per RFC 3986 at https://tools.ietf.org/html/rfc3986#section-3.3::
```

Reply:
Changed to 'purl'.